### DISCUSSION TITLE
Images won't load after connecting to MongoDB Atlas

### DISCUSSION THREAD

**Discussion by HenrikeStahlhut** [neuefische/questions#608] [Web] [answered]
Hi everyone ✌🏻, 

**Short introduction:** 
I have a database with two collections, rooms and plants. The rooms each have a `RoomType`, which assigns the respective image to each room in the overview page with the following code: 

```
<Image src={`/images/rooms/${type}.png`} width={128} height={128} alt={title} />
```
The database looks like this: 
![Screenshot 2023-05-19 at 10 44 30](https://github.com/neuefische/questions/assets/126799897/65629675-3cc2-4153-9f2d-c17819810ec7)

The images are in the [public folder](https://github.com/HenrikeStahlhut/capstone-project/tree/main/public/images/rooms) in my project and are named after the `RoomTypes`. 

**Problem:** 
Everything worked just fine when I had a local database on MongoDB Compass. After switching to MongoDB Atlas, the images aren't displayed anymore and get i get the following errors: 

![Screenshot 2023-05-19 at 10 36 20](https://github.com/neuefische/questions/assets/126799897/1c065f67-f0fa-4486-acc7-b2449d3109d6)

Except for the images, the data gets fetched correctly and everything else works. 

- [Vercel](https://capstone-project-git-feat-plants-in-room-henrikestahlhut.vercel.app/)
- [Repo](https://github.com/HenrikeStahlhut/capstone-project)

If you have any ideas/solutions/questions, hit me up! 🌼

**Comment by ThomasF85** [comment, marked as answer]
Hi Henrike,

your image files do not have capital letters in their filenames. They are named `dining_room.png` etc. You are trying to access files with capital letters in the filename, for example: `DINING_ROOM.png` => these files do not exist and cannot be found.
You are fetching all rooms from your api (which in turn probably gets the rooms from your database). The API response sends the room types in all capital letters, these room types are then used to create the path to your files, hence the error.
The problem is probably that your database contains the room types in capital letters.

**Comment by HenrikeStahlhut** [comment]
Thank you so much, it works! I thought it didn't matter if the names are upper or lowercase.

**Comment by HenrikeStahlhut** [comment]
@ThomasF85 I changed the file names to uppercase. The local host works now, vercel on the other hand still throws errors: 

![Screenshot 2023-05-19 at 11 45 36](https://github.com/neuefische/questions/assets/126799897/cf1cdfd1-97a2-4d67-a7d3-38fb1c25fe8d)

I guess it's because vercel can't access the images, but i don't really know why. I also committed after my changes and redeployed. 

Do you have any ideas?

**Comment by HenrikeStahlhut** [comment]
Update: Fixed it
